Makefile

```diff
@@ -1,6 +1,8 @@
 NAME      := MACSio
 SRC_EXT   := gz
 
+PR_REPOS := $(shell set -x; git show -s --format=%B | sed -ne 's/^PR-repos: *\(.*\)/\1/p')
+
```

Comment:
As noted, this should come from an updated daos-stack/packaging, once daos-stack/packaging#30 lands.